tools/shared.py

```diff
@@ -422,9 +425,16 @@ def __init__(self):
         atexit.register(lock.release)
 
   def get_temp_files(self):
-    return tempfiles.TempFiles(
-      tmpdir=self.TEMP_DIR if not DEBUG else get_emscripten_temp_dir(),
-      save_debug_files=os.environ.get('EMCC_DEBUG_SAVE'))
+    if DEBUG_SAVE:
+      # In debug mode store all temp files in emscripten-specific dir
+      # and don't worry about cleaning them up.
+      tmpdir = get_emscripten_temp_dir()
+      save_debug_files = True
+    else:
+      # Otherwise use the system tempdir and try to clean up after ourselves.
+      tmpdir = self.TEMP_DIR
+      save_debug_files = False
+    return tempfiles.TempFiles(tmpdir, save_debug_files)
```

Comment:
```suggestion
      return tempfiles.TempFiles(get_emscripten_temp_dir(), save_debug_files=True)
    else:
      # Otherwise use the system tempdir and try to clean up after ourselves.
      return tempfiles.TempFiles(self.TEMP_DIR, save_debug_files=False)
```

Reply:
Maybe that is simpler?